examples/digital_fingerprinting/production/morpheus/dfp/stages/multi_file_source.py

```diff
@@ -69,7 +69,7 @@ def supports_cpp_node(self):
 
     def _generate_frames_fsspec(self):
 
-        files: fsspec.core.OpenFiles = fsspec.open_files(self._filenames, filecache={'cache_storage': './.cache/s3tmp'})
+        files: fsspec.core.OpenFiles = fsspec.open_files(self._filenames)
```

Comment:
Does caching not work correctly, or did we just decide not to do it?

Reply:
Leaving this in causes a keyword argument error when using an s3 source (works for local file sources). The `filecache` arg isn't documented. 
Users wanting to use a filecache can still do so by prefixing their file urls with `filecache::`